spec/support/saml/response_builder.rb

```diff
@@ -51,14 +51,13 @@ def build_saml_response(
           )
         end
       end
-
+      # rubocop:enable Metrics/ParameterLists
```

Comment:
I don't think the changes in this file are necessary?

Reply:
Rachal Cassity asked us to put those in as they were in the code before our changes and got removed in the process of all the changes done throughout this PR.  They exist currently in the master branch.